build/docker/Dockerfile-stable

```diff
@@ -30,25 +27,20 @@ RUN apt-get update && apt-get install -q -y --fix-missing \
 RUN pip install --upgrade pip
 RUN gem install zapr
 RUN pip install zapcli
-# Install latest dev version of the python API
-RUN pip install python-owasp-zap-v2.4
```

Comment:
Oh I see… In this case, we should run this command before zapcli, to avoid a useless command